CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta13 (in progress)
+==========================
+
+New Features
+------------
+
+- In a **Group Room**, you can now have more control over how your available bandwidth
+  is distributed among the RemoteVideoTracks that you have subscribed to. twilio-video.js
+  introduces the [Bandwidth Profile API](TODO(mmalavalli)). Note that this feature is
+  currently in **private beta** and hence will be **opt-in**. Please reach out to
+  [email@address.com](TODO(mmalavalli)) for more information about how to enable these
+  APIs for your Twilio Account. **Using these APIs in a Peer-to-Peer Room will have no effect**.
+  
+  ### Track Priority
+
+  While publishing a LocalTrack, you can now optionally specify its publish priority
+  in the following way:
+
+  ```js
+  const localTrackPublication = await room.localParticipant.publishTrack(localTrack, {
+    priority: 'high' // LocalTrack's publish priority - "low", "standard" or "high"
+  });
+
+  // LocalTrackPublication has a new property "priority" which stores the publish
+  // priority set while publishing the corresponding LocalTrack.
+  assert.equal(localTrackPublication.priority, 'high');
+  ```
+
+  This signals to the media server that this LocalTrack is more important than other
```

Comment:
The track priority can be used to indicate that a track is more important and also less important. I would not constraint the explanation to the 'high' case. In addition, the current explanation seems to make understand that priorities work "per-participant". Hence, I would be saying instead something like "This signals to the media server the relevance of a track with respect to other tracks published to the Room"